hieradata/nodes/hozer-65.yaml

```diff
@@ -0,0 +1,2 @@
+classes:
```

Comment:
you should probably rename this and promote hozer-65 to a prod vm

Reply:
@kkuehler don't forget to make this a proper VM before merging

Reply:
Also when you make it a proper VM please make sure to change its IP to be something less than 65; I'd like the start of the hozer range to stay as 65.